lib/components/viewers/stop-viewer.js

```diff
@@ -248,7 +246,7 @@ class StopViewer extends Component {
               id: 'components.StopViewer.zoomToStop'
             })}
           >
-            <StyledIconWrapper>
+            <StyledIconWrapper style={{ marginLeft: '0.2em' }}>
```

Comment:
There are a few instances like this one where you could write `<Icon Icon={Search} style=... />`